x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/planner/QueryTranslator.java

```diff
@@ -845,6 +818,14 @@ public QueryTranslation translate(Expression exp, boolean onAggs) {
 
         protected abstract QueryTranslation asQuery(E e, boolean onAggs);
 
+
+        protected static Query generateQuery(ScalarFunction sf, Expression field, Supplier<Query> query) {
```

Comment:
maybe better suggestion for a name here?

Reply:
`wrapQuery`/`handleQuery`/`handleScript`?